pkg/labels/selector.go

```diff
@@ -809,3 +809,77 @@ func SelectorFromSet(ls Set) Selector {
 func ParseToRequirements(selector string) ([]Requirement, error) {
 	return parse(selector)
 }
+
+// Conflicts takes 2 maps
```

Comment:
This also only checks the keys in the first map. If you have keys in the second map that don't exist in the first, this function doesn't consider that a conflict.